app/_src/gateway-operator/support.md

```diff
@@ -17,34 +17,10 @@ LTS versions of {{site.kgo_product_name}} are supported for 3 years after releas
 {:.note}
 > {{site.kgo_product_name}} is a recently released product and does not currently provide an LTS version.
 
-## Supported versions
-
-{% assign latest_kgo_release = site.data.kong_latest_kgo.release %}
-{% assign latest_kgo_eol = "Sept 2024" %}
-
-<table style="display:table" width="100%">
-<thead>
-<tr>
-  <th>Kubernetes Version</th>
```

Comment:
I agree that we should remove this column from the support page. Please make sure https://deploy-preview-7325--kongdocs.netlify.app/gateway-operator/latest/reference/version-compatibility/ is linked and very easy to find from this page